svc/pkg/mm/worker/redis-scripts/lobby_find.lua

```diff
@@ -11,6 +11,12 @@ local key_find_query_player_ids = KEYS[2]
 local key_ns_player_ids = KEYS[3]
 local key_player_unregistered = KEYS[4]
 
+-- Extract names into list
+local lobby_tag_names = {}
```

Comment:
imo `lobby_tag_names` is misleading. Maybe `query_tag_names`?

Reply:
It's meant to be "the names of the lobby tags" as in the name of the lobby tag `my_tag: foo` would be `my_tag`

Reply:
https://github.com/rivet-gg/rivet/pull/197/commits/4d08c9ca1fe72741435e436e5e7aa8426c25943d